app/views/pages/codebar-stories-podcast.html.haml

```diff
@@ -0,0 +1,20 @@
+.div.mb-5
+    = image_tag 'codebar-stories-banner.png', alt: 'codebar stories banner', class: 'mw-100'
```

Comment:
@KimberleyCook can we convert the alt text into s translation string as well? Going forward alt texts should go into the translation .yml files as well.

Reply:
Ok, all done now